pkg/client/clientset_generated/release_1_5/typed/authorization/v1beta1/authorization_client.go

```diff
@@ -25,6 +25,8 @@ import (
 
 type AuthorizationInterface interface {
 	GetRESTClient() *restclient.RESTClient
+	LocalSubjectAccessReviewsGetter
```

Comment:
I'm looking into this right now. It appears that the answer is no. I've sent out https://github.com/kubernetes/kubernetes/pull/32720